CODEOWNERS

```diff
@@ -1,8 +1,8 @@
-* @sanpj2292 @saikumarrs @arnab-p @sivashanmukh @utsabc @ItsSudip @shrouti1507
+* @rudderlabs/integrations-platform @arnab-p @sivashanmukh
 __tests__/ @rudderlabs/integrations
 __mocks__/ @rudderlabs/integrations
 v0/ @rudderlabs/integrations
+cdk/ @rudderlabs/integrations
 features.json/ @rudderlabs/integrations
```

Comment:
integrations team will be adding new destinations to Proxy & will update `adapters/networkHandlerFactory.js` - so shouldn't it be as-is ?

Reply:
My bad. Created a [new PR](https://github.com/rudderlabs/rudder-transformer/pull/1541) to avoid this dependency. Plz, review.